### PR TITLE
Add Service model connected to Record & Domain 

### PR DESCRIPTION
**Pull request by ghost**
Record/Domain ownership is being delegated to Service Owner.
This introduces models to this change.

### CODE REVIEW

File: powerdns/models/ownership.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+class OwnershipType(Enum):
+    BO = 'Business Owner'
+    TO = 'Technical Owner'
+
+
+class Service(TimeTrackable):
+    name = models.CharField(
+        _("name"),
+        unique=True,
```
Comment (szok) [inline powerdns/models/ownership.py]: are you sure if unique?

File: powerdns/models/ownership.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    PLANNING = 'Planning'
+    PENDING = 'Pending'
+
+
+class OwnershipType(Enum):
+    BO = 'Business Owner'
+    TO = 'Technical Owner'
+
+
+class Service(TimeTrackable):
```
Comment (szok) [inline powerdns/models/ownership.py:33]: add **str** method :)

File: powerdns/models/ownership.py (hunk 1 above)
Comment (ghost) [reply powerdns/models/ownership.py]: I'll rm it because there is also `uid` which is `unique`.

**Review by ar4s — APPROVED** [review]

**Review by szok — APPROVED** [review]

File: powerdns/models/ownership.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+from enum import Enum
+
+from django.conf import settings
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from powerdns.utils import TimeTrackable
+
+
+class ServiceStatus(Enum):
```
Comment (mkurek) [inline powerdns/models/ownership.py:20]: (i know it's already merged but..) we don't need so many statuses - in fact we only need to know if service is active or not (simple bool field) and leave the logic behind these statuses elsewhere

**Pull request merged**